qcore/simulation_structure.py

```diff
@@ -8,7 +8,7 @@
 
 def get_fault_from_realisation(realisation):
     realisation = os.path.basename(realisation)  # if realisation is a fullpath
-    return realisation.split("_")[0]
+    return realisation.split("_REL")[0]
```

Comment:
Can we change this to rsplit("_REL", 1)[0]?

```
In [6]: test                                                                                        
Out[6]: 'MT_RELERSON_REL01'

In [7]: test.rsplit("_REL",1)[0]                                                                    
Out[7]: 'MT_RELERSON'
```

I'm not 100% convinced that other parts of the workflow can handle underscores in the fault name but we can cross that bridge later.